apps/explorer/src/app/components/blocks/home/blocksTable.tsx

```diff
@@ -0,0 +1,52 @@
+import { TendermintBlockchainResponse } from '../../../routes/blocks/tendermint-blockchain-response';
```

Comment:
Prefer `kebab-casing` for file names over `camelCase` or `PascalCase`, there's some other files kicking around with the same issue:

* txContent -> tx-content 
* txDetails -> tx-detials
* ect..

Reply:
will do!